src/dbus_api/api/manager_2_0/methods.rs

```diff
@@ -36,7 +36,8 @@ pub fn create_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
     let object_path = m.path.get_name();
     let dbus_context = m.tree.get_data();
     let mut engine = dbus_context.engine.borrow_mut();
-    let result = engine.create_pool(name, &blockdevs, tuple_to_option(redundancy));
+
+    let result = pre_pool_op! {"preparing to create pool with name {}", name; "successfully created pool with name {}, uuid {}", name; engine.create_pool(name, &blockdevs, tuple_to_option(redundancy))};
```

Comment:
Please use the macro!() form as opposed to the macro!{} form. While these are identical from a code compilation perspective, the convention is usually to use the macro!() form for something that acts vaguely like a function. I'd argue that this acts more like a function than another construct as it simply adds some logging around the call we pass in.

Reply:
Resolved.